superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -240,6 +240,13 @@ export const controls = {
     description: 'Sort bars by x labels.',
   },
 
+  combine_metric: {
+    type: 'CheckboxControl',
+    label: 'Combine Metrics',
+    default: false,
+    description: 'Combine Metrics to display.',
```

Comment:
Oh I understand, we need a clearer description here. `Display metrics side by side within each column, as opposed to each column being displayed side by side for each metric.`